master/buildbot/test/fake/fakedb.py

```diff
@@ -528,6 +533,7 @@ class LogChunk(Row):
         compressed=0)
 
     required_columns = ('logid', )
+    utf8_columns = ('content', )
```

Comment:
Thanks it is clear now.

I think it is worth to put this explaination as a comment in the code.